geonetwork/transform-mcp-iso19115-3/src/test/java/au/org/aodn/xslt/TransformCatalogueTest.java

```diff
@@ -96,6 +96,16 @@ public void testAcquisitionInformation() throws IOException {
         testFiles("acquisitionInformation");
     }
 
+    @Test
+    public void testSamplingFrequency() throws IOException {
+        testFiles("samplingFrequency");
```

Comment:
Fails to run as directory is currently samplingfrequency

Reply:
Fixed

Reply:
Still failing export directory also needs to be renamed